.github/workflows/python-publish.yml

```diff
@@ -1,18 +1,21 @@
 # This workflow will
 # - build distribution package, pure python wheel
-# - deploy produced distribution package to PyIP
+# - publish produced distribution package to PyPI
 #
 # For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
 
-name: Deploy Python Package
+name: Publish Python Package
 
+# Controls when the workflow will run
 on:
-  workflow_dispatch:
```

Comment:
Why was `workflow_dispatch` removed? This allows manual rerunning of the pipeline in case something didn't work with the publish step without needing to artificially change to code for pushing.

Reply:
It is still possible it was just moved down